src/shared/components/sectionHeader/SectionHeader.tsx

```diff
@@ -28,10 +28,12 @@ export default class SectionHeader extends React.Component<ISectionHeaderProps,
 		let {promises, children, className, secondaryComponent, ...divProps} = this.props;
 		return (
 			<div className={classNames(className, styles.SectionHeader)} {...divProps}>
-				{!!(promises && promises.some(promise => promise.isPending)) && (
-					<Spinner/>
-				)}
-				<h2>{children}</h2>
+				<h2>
+					{children}
+					{!!(promises && promises.some(promise => promise.isPending)) && (
+						<ThreeBounce style={{ display:'inline-block', marginLeft:10 }}/>
```

Comment:
Using the `ThreeBounce` component directly here instead of `LoadingIndicator` in order to get the spinner on the same line with the header text.